libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -390,7 +332,7 @@ void AC_AutoTune::run()
             last_pilot_override_warning = now;
         }
     }
-    if (zero_rp_input) {
+    if (zero_rp_input && !allow_pilot_rp_input()) {
```

Comment:
This allows the pilot to make inputs while the automated testing is being conducted.  Some of the tests are 20 seconds long and the pilot is allowed to make small inputs to help maintain position during the automated Angle P test.  The multicopter autotune was designed to automatically make inputs in between twitches to help maintain position, if autotune was entered from a GPS aided mode.  Mine is different because it is doing it during the automated test.  Now that I have shown that the rate P and rate D tests are being held by velocity feedback, I may remove the pilot input and implement the same code for the angle P test.

Reply:
Removed pilot input during ANGLE_P tuning and added velocity feedback.